torch/utils/cpp_extension.py

```diff
@@ -201,6 +202,7 @@ def _join_rocm_home(*paths) -> str:
 ROCM_HOME = _find_rocm_home()
 HIP_HOME = _join_rocm_home('hip') if ROCM_HOME else None
 IS_HIP_EXTENSION = True if ((ROCM_HOME is not None) and (torch.version.hip is not None)) else False
+IS_XPU_EXTENSION = True if torch.version.xpu is not None else False
```

Comment:
"EXTENSION" does not sound right. XPU is official backend in PyTorch, not extension. Was naming inherited from IPEX?

Reply:
Should be OK.
Here the EXTENSION is used to distinguish the build target from CUDA/HIP/.. EXTENSION, instead of XPU itself?

Reply:
@gujinghui @dvrogozh Here the EXTENSION is to distinguish the build target, the cpp extension is build for XPU, like CUDA/XIP, it does not meantion to ipex.